content/docs/04-clusters/03-edge/04-edgeforge-workflow/12-build-artifacts.md

```diff
@@ -0,0 +1,309 @@
+
+  Fill in the required fields, such as the API key name and expiration date, and confirm your changes. Copy the key value to your clipboard to use in the next step.
+
+  ![Screenshot of generating an API key in Palette.](/tutorials/deploy-pack/registries-and-packs_deploy-pack_generate-api-key.png )
+
+  
+
+14. Set the API key by issuing the following command. Replace the `[USE-YOUR-API-KEY_HERE]` placeholder with your API key. The Palette Edge CLI will use this API key to authenticate itself when interacting with the Spectro Cloud API. Once authenticated, the Palette Edge CLI can access and manage resources in your Spectro Cloud account.
+<br />
+
+  ```bash
+  export API_KEY=[USE-YOUR-API-KEY_HERE]
+  ```
+
+
+15. Copy the cluster profile and project IDs from Palette to use in the next step. To locate these IDs, switch to [Palette](https://console.spectrocloud.com). Navigate to the left **Main Menu** and select **Profiles**. Click on the specific cluster profile to view its details. The cluster profile and project IDs are in the cluster details page URL. The cluster details page URL follows the `[Palette-URL]/projects/[PROJECT-ID]/profiles/cluster/[PROFILE-ID]` syntax. 
+
+  For example, the screenshot below highlights the cluster profile and project IDs in a cluster details page URL. 
+
+  ![A screenshot highlighting the cluster profile and project IDs in the URL of the cluster details page.](/clusters_edge_edge-forge-workflow_build-images_build-artifacts_url.png)
+
+
+
+16. Set the profile ID using the following command. Replace the `[USE-YOUR-PROFILE-ID_HERE]` placeholder with your cluster profile ID. This command sets the profile ID, which identifies the specific cluster profile that you want to use for creating the offline content bundle. 
+<br />
+
+  ```bash
+  export PROFILE_ID=[USE-YOUR-PROFILE-ID_HERE]
+  ```
+
+
+17. Set the project ID by issuing the following command. Replace the `[USE-YOUR-PROJECT-ID_HERE]` placeholder with your cluster profile ID.
+The offline content bundle will be associated with the project ID you will use in the following command. 
+<br />
+
+  ```bash
+  export PROJECT_ID=[USE-YOUR-PROJECT-ID_HERE]
+  ```
+
+
+18. Build the offline content bundle using the following command. The content bundle will have the configuration and settings for the Edge cluster, including the operating system image, Kubernetes distribution, any additional packs, and manifest files specified in your cluster profile you wish to deploy to the Edge cluster.
+
+  The command below uses the palette-edge tool to build the offline content bundle. The `--api-key`, `--project-id`, and `--cluster-profile-ids` options specify the necessary parameters for the build process. The `--palette-endpoint` option specifies the endpoint URL for the Spectro Cloud API. The `--outfile` option specifies the name of the output file for the content bundle.
```

Comment:
```suggestion
18.  The `build` command creates the content bundle However, the command requires a set of parameters to start. The `--api-key`, `--project-id`, `--cluster-profile-ids`, and  `--palette-endpoint` are required flags.  Use the `--outfile` option to specify the name of the output file for the content bundle. Issue the following command to start the content bundle build process.
```

We can simplify this. Remember, you mentioned this in the intro (see below) so you don't have to repeat yourself here. You emphasize this in the conclusion but it's necessary here. 

_A content bundle is an archive that includes the operating system image, Kubernetes distribution, additional layers, and packs and manifest files specified in your cluster profile that you want to deploy to the Edge cluster. A content bundle provides several benefits, such as:_